### ISSUE TITLE
Really long lines, CSS animation moves super fast

### ISSUE THREAD

**Issue by mhulse** [beekman/Rainbow-Selection-Flux#12]
I'd like for it to be a constant speed no matter the length of the line.

**Comment by beekman** [comment]
Agreed- I just didn't think it was something we could easily control. What would you recommend?

**Comment by mhulse** [comment]
> Agreed- I just didn't think it was something we could easily control. What would you recommend?

Hey! Yah, I would have to play with it. Pixel values could work, but, IIRC, I liked percent better for smaller sizes.

I'm pretty sure pixel values would fix … Maybe breakpoints? Shit, hard to say as some lines are super short and some long.

Would have to play with it in order to give my best suggestion. :(
